### PR TITLE
Add Changelog to setup.cfg

### PR DESCRIPTION
**Pull request by adamchainz**
One of the main reasons I hit up a project's PyPI page is to get to its changelog. Adding a changelog in the sidebar gets a nice present icon, see e.g. https://pypi.org/project/pytest-randomly/

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -26,6 +26,7 @@ classifiers =
     Topic :: Utilities
 keywords = test, unittest
 project_urls =
+    Changelog=https://docs.pytest.org/en/latest/changelog.html
```
Comment (bluetech) [inline setup.cfg:29]: Perhaps `/stable` would be better? Not sure...

**Review by bluetech — APPROVED** [review]
Thanks, looks handy to me.

File: setup.cfg (hunk 1 above)
Comment (adamchainz) [reply setup.cfg:29]: As I understand, the 'latest' page always includes the entire changelog.

Comment (nicoddemus) [reply setup.cfg:29]: The latest page on PyPI is about the last release, correct? If so the correct changelog would be `stable`: https://docs.pytest.org/en/stable/changelog.html

Comment (adamchainz) [inline setup.cfg:29]:
```suggestion
    Changelog=https://docs.pytest.org/en/stable/changelog.html
```

Comment (adamchainz) [reply setup.cfg]: Okay fair

File: setup.cfg [hunk 2]
```diff
@@ -26,6 +26,7 @@ classifiers =
     Topic :: Utilities
 keywords = test, unittest
 project_urls =
+    Changelog=https://docs.pytest.org/en/stable/changelog.html
```
Comment (graingert) [inline setup.cfg:29]: there's a full list of all the special icons warehouse supports:

https://github.com/pypa/warehouse/blob/851e4c37bc1a9d1ef299e9300c7aa3ffb7cb09d5/warehouse/templates/packaging/detail.html#L20-L58

we may want to add some more eg https://twitter.com/pytestdotorg

**Review by graingert — APPROVED** [review]

File: setup.cfg (hunk 2 above)
Comment (nicoddemus) [reply setup.cfg:29]: Should it be then: `twitter=https://twitter.com/pytestdotorg`?

Could you please add an inline suggestion if you are sure about the syntax @graingert ?

**Review by nicoddemus — APPROVED** [review]
Thanks @adamchainz!

File: setup.cfg (hunk 2 above)
Comment (graingert) [reply setup.cfg:29]:
```suggestion
    Changelog=https://docs.pytest.org/en/stable/changelog.html
    Twitter=https://twitter.com/pytestdotorg
```

@nicoddemus do we want the coverage on here? or anything else from that list?

Comment (nicoddemus) [reply setup.cfg:29]: IMHO this is good enough... no need to add coverage/CI information here, I think.

**Comment by nicoddemus** [comment]
Backport: https://github.com/pytest-dev/pytest/pull/8167

**Pull request merged**